src/main/java/org/broadinstitute/hellbender/tools/coveragemodel/CopyRatioHiddenMarkovModelResults.java

```diff
@@ -0,0 +1,49 @@
+package org.broadinstitute.hellbender.tools.coveragemodel;
+
+import org.broadinstitute.hellbender.tools.exome.Target;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.hmm.ForwardBackwardAlgorithm;
+
+import javax.annotation.Nonnull;
+import java.io.Serializable;
+import java.util.List;
+
+/**
+ * This class stores the results of forward-backward and Viterbi algorithms for HMM-based
+ * copy ratio (or copy number) models.
+ *
+ * @param <D> copy ratio emission data type
+ * @param <S> type of hidden state
+ *
+ * @author Mehrtash Babadi &lt;mehrtash@broadinstitute.org&gt;
+ */
+public final class CopyRatioHiddenMarkovModelResults<D, S> implements Serializable {
```

Comment:
@davidbenjamin I used `D`, `S`, and `T` for concordance with `HiddenMarkovModel`. I'll accept/apply your suggestion though.